ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/product/v1/product/quantity.html

```diff
@@ -12,6 +12,7 @@
   ~ 
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
 <template data-sly-template.quantity="${ @ product, allowZeroQuantity, defaultQuantity}">
+    <h2 class="productFullDetail__quantityTitle option__title"><span>${'Quantity' @ i18n}</span></h2>
```

Comment:
This is going to break the layout of grouped products where the "quantity" title is not displayed. This should not be displayed for grouped products.

Reply:
Oh you are right, I move it back to the `product-html` template. I only tested with normal products.